crates/streamer/Cargo.toml

```
@@ -21,6 +21,7 @@ webm-iterable = { version = "0.6", features = ["futures"] }
 cadeau = { version = "0.5", features = ["dlopen"] }
 thiserror = "2"
 num_cpus = "1.16.0"
+serde_json = "1.0.133"
```

suggestion: Avoid to over-constrain dependencies: 
```suggestion
num_cpus = "1.16"
serde_json = "1.0"
```

---

updated, removed serde json, it was used in an experiment in an discarded commit